core/resources/shaders/point.vs

```diff
@@ -26,11 +18,19 @@ uniform float u_meters_per_pixel;
 
 #pragma tangram: uniforms
 
-varying float v_alpha;
-varying vec2 v_texcoords;
+attribute vec2 a_uv;
+attribute LOWP vec2 a_position;
+attribute LOWP vec2 a_screenPosition;
+attribute LOWP float a_alpha;
+attribute LOWP float a_rotation;
+attribute LOWP vec4 a_color;
+attribute LOWP vec4 a_stroke;
+
 varying vec4 v_color;
```

Comment:
Yeah we should remove the two first qualifiers, and add `lowp` hint to `a_uv`, this could give bad behavior when driver uses this hint on the position attribute, even if they are in a restricted range in that case.